File: PHP/Events/GetEventsAvailableTicketsResult.php
<?php
namespace LTD_API;

class GetEventsAvailableTicketsResult
{
    //array of AvailableEventInfo
    private $availableEvents;
    
    public function getAvailableEvents()
    {
        return $this->availableEvents;
    }
    public function setAvailableEvents($availableEvents)
    {
        $this->availableEvents = $availableEvents;
    }
}

File: Java/Events/GetEventsAvailableTicketsResult.java
package LTD_API;

import java.util.List;

public class GetEventsAvailableTicketsResult
{
    private List<AvailableEventInfo> availableEvents;
    
    public List<AvailableEventInfo> getAvailableEvents()
    {
        return availableEvents;
    }
    public void setAvailableEvents(List<AvailableEventInfo> availableEvents)
    {
        this.availableEvents = availableEvents;
    }
}

